src/Table/AbstractTable.php

```diff
@@ -263,10 +263,16 @@ public function delete(RowInterface $row)
      * @return RowInterface
      *
      */
-    public function newRow(array $cols = [])
+    public function newRow(array $colsVals = [])
```

Comment:
> I thought you might be @yespire :-)

lol. no. I am not. I understand I wrote `I am` , what I was supposed to write was `I and` . Yes we also have not found a better naming / wording that is the reason we have been waiting to open an issue. Thank you for your consideration.